tsfile/src/main/java/org/apache/iotdb/tsfile/read/expression/QueryExpression.java

```diff
@@ -20,11 +20,13 @@
 
 import java.util.ArrayList;
 import java.util.List;
+import org.apache.iotdb.tsfile.file.metadata.enums.TSDataType;
 import org.apache.iotdb.tsfile.read.common.Path;
 
 public class QueryExpression {
```

Comment:
This class is no longer used in the main code. You should either reuse it or remove it.